docs/_snippets/installation/plugins-mapping/plugins-mapping.html

```diff
@@ -0,0 +1,848 @@
+            </td>
+            <td>N/A</td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/easyimage">easyimage</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/images/image-upload/easy-image.html">Easy Image</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/editorplaceholder">editorplaceholder</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/editor-placeholder.html">Editor placeholder</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/elementspath">elementspath</a></p>
+            </td>
+            <td>N/A</td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/embed">embed</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/media-embed.html">Media embed</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/embedbase">embedbase</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/media-embed.html#media-providers">Media embed config</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/embedsemantic">embedsemantic</a></p>
+            </td>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/docs/ckeditor5/latest/features/media-embed.html#semantic-data-output-default">Media embed config</a></p>
+            </td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/emoji">emoji</a></p>
+            </td>
+            <td>N/A</td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/enterkey">enterkey</a></p>
+            </td>
+            <td>N/A</td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/entities">entities</a></p>
+            </td>
+            <td>N/A</td>
+         </tr>
+         <tr>
+            <td>
+               <p style="margin-left:0px;"><a href="https://ckeditor.com/cke4/addon/fakeobjects">fakeobjects</a></p>
+            </td>
+            <td>N/A</td>
```

Comment:
I'd say this could be replaced by a custom [widget](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/tutorials/implementing-a-block-widget.html).